web/src/unread_ops.ts

```diff
@@ -256,31 +258,71 @@ export function mark_as_unread_from_here(
     }
     message_lists.current.prevent_reading();
 
-    // If we have already fully fetched the current view, we can
-    // send the server the set of IDs to update, rather than
-    // updating on the basis of the narrow.
-    let message_ids_to_update;
-    if (message_lists.current.data.fetch_status.has_found_newest()) {
-        message_ids_to_update = message_lists.current
-            .all_messages()
-            .filter(
-                (msg) =>
-                    (include_anchor && msg.id >= message_id) ||
-                    (!include_anchor && msg.id > message_id),
-            )
-            .map((msg) => msg.id);
+    const rounding_factor = 25;
```

Comment:
Added these as constants, but found other values like the 200 (in `do_mark_unread()`) hard-coded in the same function, so I'm not sure if this is overdone.

Reply:
Is there a CZO discussion on this? I think it would be helpful to start one to discuss these values with folks on this.
If there's already an CZO discussion please do link it to the PR description.

Reply:
The CZO discussion is linked in the first line of the issue that this PR solves.
I've now edited the PR description to include the CZO link now, like you requested.

Reply:
Constants near the top of the file, a la `INITIAL_BATCH_SIZE`, are often good, but generally require more specific names; `threshold_fetched` is probably too generic for that.